src/lib.rs

```diff
@@ -86,7 +86,6 @@ impl Default for WasiSockaddr {
 #[link(wasm_import_module = "wasi_snapshot_preview1")]
 extern "C" {
     pub fn sock_open(addr_family: u8, sock_type: u8, fd: *mut u32) -> u32;
-    pub fn sock_close(fd: u32);
```

Comment:
Please keep this function, it will be handled in the near future.

Reply:
Got and fixed it.